db/migrate/20230815113642_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :users, null: false, foreign_key: true
```

Comment:
As per the ER diagram provided to you, the relation between the comments and the user is set up through the author just like for the posts. Kindly update your code just like you did for the posts.